cmd/exporters/influxdb/influxdb_test.go

```diff
@@ -6,27 +6,36 @@ package influxdb
 import (
 	"goharvest2/cmd/poller/exporter"
 	"goharvest2/cmd/poller/options"
+	"goharvest2/pkg/conf"
 	"goharvest2/pkg/matrix"
-	"goharvest2/pkg/tree/node"
 	"testing"
 )
 
 // test that the addr (and port) parameters
 // are handled properly to construct server URL
 func TestAddrParameter(t *testing.T) {
-
 	expectedURL := "http://localhost:8086/api/v2/write?org=netapp&bucket=harvest&precision=s"
 
 	opts := &options.Options{}
 	opts.Debug = true
+	var params map[string]conf.Exporter
+	exporterName := "influx-test-addr"
 
-	params := node.NewS("")
-	params.NewChildS("addr", "localhost")
-	params.NewChildS("org", "netapp")
-	params.NewChildS("bucket", "harvest")
-	params.NewChildS("token", "xxxxxxx")
+	path := "../../tools/doctor/testdata/testConfig.yml"
+	err := conf.LoadHarvestConfig(path)
```

Comment:
not needed since you're calling  `conf.GetExporters2` below

Reply:
done